src/rate/demos/enUS/disabled.demo.md

```diff
@@ -0,0 +1,5 @@
+# Read only
```

Comment:
`readonly` and `disabled` are 2 different concept. In the PR, readonly prop is better than disabled.

Compared with `readonly`, `disabled` awalys means different looking.